### PR TITLE
f-checkout@v0.72.0 - replace city with locality

### PR DESCRIPTION
**Pull request by FayeCarter**
### Changed
- `address.city` to `address.locality` in checkout state.
- `UPDATE_STATE` mutation to use `locality`.
- `getAddress` to use locality.

### CODE REVIEW

File: packages/components/organisms/f-checkout/src/store/tests/checkout.module.test.js [hunk 1]
```diff
@@ -347,19 +357,33 @@ describe('CheckoutModule', () => {
                 };
 
                 axios.get = jest.fn(() => Promise.resolve({ data: customerAddresses }));
-                const [expectedAddress] = customerAddresses.Addresses;
+
+                mapGetAddressRequestSpy = jest.spyOn(mapper, 'mapGetAddressRequest').mockImplementation(() => expectedAddress);
+            });
+
+            it('should call `mapGetAddressRequest` with address.', async () => {
```
Comment (FayeCarter) [inline packages/components/organisms/f-checkout/src/store/tests/checkout.module.test.js:364]: New Test

File: packages/components/organisms/f-checkout/src/services/tests/mapper.test.js [hunk 2]
```diff
@@ -168,6 +163,21 @@ describe('mapAnalyticsName :: ', () => {
     });
 });
 
+describe('mapGetAddressRequest :: ', () => {
```
Comment (FayeCarter) [inline packages/components/organisms/f-checkout/src/services/tests/mapper.test.js:166]: New Test

File: packages/components/organisms/f-checkout/src/services/mapper.js [hunk 3]
```diff
@@ -56,6 +56,12 @@ const mapUpdateCheckoutRequest = ({
     }
 ]);
 
+const mapGetAddressRequest = address => {
```
Comment (FayeCarter) [inline packages/components/organisms/f-checkout/src/services/mapper.js:59]: This is new

File: packages/components/organisms/f-checkout/src/services/mapper.js [hunk 4]
```diff
@@ -56,6 +56,15 @@ const mapUpdateCheckoutRequest = ({
     }
 ]);
 
+/**
+ * Replaces `city` key with `locality` key
+ */
+const mapCityToLocality = address => {
```
Comment (FayeCarter) [inline packages/components/organisms/f-checkout/src/services/mapper.js:62]: This is new

File: packages/components/organisms/f-checkout/src/services/tests/mapper.test.js [hunk 5]
```diff
@@ -168,6 +163,21 @@ describe('mapAnalyticsName :: ', () => {
     });
 });
 
+describe('mapCityToLocality :: ', () => {
```
Comment (FayeCarter) [inline packages/components/organisms/f-checkout/src/services/tests/mapper.test.js:166]: New test

File: packages/components/organisms/f-checkout/src/store/tests/checkout.module.test.js [hunk 6]
```diff
@@ -347,19 +357,33 @@ describe('CheckoutModule', () => {
                 };
 
                 axios.get = jest.fn(() => Promise.resolve({ data: customerAddresses }));
-                const [expectedAddress] = customerAddresses.Addresses;
+
+                mapCityToLocalitySpy = jest.spyOn(mapper, 'mapCityToLocality').mockImplementation(() => expectedAddress);
+            });
+
+            it('should call `mapCityToLocality` with address.', async () => {
```
Comment (FayeCarter) [inline packages/components/organisms/f-checkout/src/store/tests/checkout.module.test.js:364]: New test

**Review by charliearlie — DISMISSED** [review]

**Review by charliearlie — APPROVED** [review]

File: packages/components/organisms/f-checkout/src/tenants/en-GB.js [hunk 7]
```diff
@@ -6,7 +6,7 @@ const messages = {
         addressGroup: 'Address',
         line1: 'Address line 1',
         line2: 'Address line 2 (optional)',
-        city: 'City',
+        locality: 'City',
```
Comment (SachaZvetelman) [inline packages/components/organisms/f-checkout/src/tenants/en-GB.js:9]: Ultra nit: if you want, change this to `Locality`. Still, approving the PR.

**Review by SachaZvetelman — APPROVED** [review]

**Pull request merged**